packages/core/src/utils/__tests__/shortenedUrl.test.ts

```diff
@@ -0,0 +1,114 @@
+  status: 200,
+  ok: true,
+  url: 'https://wonderful-rabbit-5.tun2.indiciotech.io?oob=eyJAdHlwZSI6ICJkaWQ6c292OkJ6Q2JzTlloTXJqSGlxWkRUVUFTSGc7c3BlYy9vdXQtb2YtYmFuZC8xLjAvaW52aXRhdGlvbiIsICJAaWQiOiAiNzY0YWYyNTktOGJiNC00NTQ2LWI5MWEtOTI0YzkxMmQwYmI4IiwgImxhYmVsIjogIkFsaWNlIiwgImhhbmRzaGFrZV9wcm90b2NvbHMiOiBbImRpZDpzb3Y6QnpDYnNOWWhNcmpIaXFaRFRVQVNIZztzcGVjL2Nvbm5lY3Rpb25zLzEuMCJdLCAic2VydmljZXMiOiBbImRpZDpzb3Y6TXZUcVZYQ0VtSjg3dXNMOXVRVG83diJdfQ====',
+} as Response
+
+mockedResponseOobUrl['headers'] = dummyHeader
+
+const mockedResponseConnectionJson = {
+  status: 200,
+  ok: true,
+  json: async () => ({
+    '@type': 'did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/connections/1.0/invitation',
+    '@id': '20971ef0-1029-46db-a25b-af4c465dd16b',
+    label: 'test',
+    serviceEndpoint: 'http://sour-cow-15.tun1.indiciotech.io',
+    recipientKeys: ['5Gvpf9M4j7vWpHyeTyvBKbjYe7qWc72kGo6qZaLHkLrd'],
+  }),
+} as Response
+
+mockedResponseConnectionJson['headers'] = header
+
+const mockedResponseConnectionUrl = {
+  status: 200,
+  ok: true,
+  url: 'http://sour-cow-15.tun1.indiciotech.io?c_i=eyJAdHlwZSI6ICJkaWQ6c292OkJ6Q2JzTlloTXJqSGlxWkRUVUFTSGc7c3BlYy9jb25uZWN0aW9ucy8xLjAvaW52aXRhdGlvbiIsICJAaWQiOiAiMjA5NzFlZjAtMTAyOS00NmRiLWEyNWItYWY0YzQ2NWRkMTZiIiwgImxhYmVsIjogInRlc3QiLCAic2VydmljZUVuZHBvaW50IjogImh0dHA6Ly9zb3VyLWNvdy0xNS50dW4xLmluZGljaW90ZWNoLmlvIiwgInJlY2lwaWVudEtleXMiOiBbIjVHdnBmOU00ajd2V3BIeWVUeXZCS2JqWWU3cVdjNzJrR282cVphTEhrTHJkIl19',
+} as Response
+
+mockedResponseConnectionUrl['headers'] = dummyHeader
+
+let outOfBandInvitationMock: OutOfBandInvitation
+let connectionInvitationMock: ConnectionInvitationMessage
+let connectionInvitationToNew: OutOfBandInvitation
+
+beforeAll(async () => {
+  outOfBandInvitationMock = await JsonTransformer.fromJSON(mockOobInvite, OutOfBandInvitation)
+  await MessageValidator.validateSync(outOfBandInvitationMock)
+  connectionInvitationMock = await JsonTransformer.fromJSON(mockConnectionInvite, ConnectionInvitationMessage)
+  await MessageValidator.validateSync(connectionInvitationMock)
+  connectionInvitationToNew = convertToNewInvitation(connectionInvitationMock)
+})
+
+describe('shortUrlOobJson', () => {
+  test('oobToJson', async () => {
+    const short = await fromShortUrl(mockedResponseOobJson)
+    expect(short).toEqual(outOfBandInvitationMock)
+  })
+})
+describe('shortUrlOobUrl', () => {
+  test('oobFromUrl', async () => {
+    const short = await fromShortUrl(mockedResponseOobUrl)
+    expect(short).toEqual(outOfBandInvitationMock)
+  })
+})
+describe('shortUrlConnectionJson', () => {
+  test('connectionInvitationToJson', async () => {
+    const short = await fromShortUrl(mockedResponseConnectionJson)
+    expect(short).toEqual(connectionInvitationToNew)
+  })
+})
+describe('shortUrlConnectionUrl', () => {
+  test('connectionFromUrl', async () => {
+    const short = await fromShortUrl(mockedResponseConnectionUrl)
+    expect(short).toEqual(connectionInvitationToNew)
+  })
+})
```

Comment:
can we make the descriptions of the tests a bit more descriptive? I find it hard to understand what each of the different tests does